app/src/main/java/org/fossasia/openevent/general/search/SearchTimeViewModel.kt

```diff
@@ -6,6 +6,11 @@ import org.fossasia.openevent.general.data.Preference
 class SearchTimeViewModel(private val preference: Preference) : ViewModel() {
     private val tokenKeyDate = "DATE"
     private val tokenKeyNextDate = "NEXT_DATE"
+    private val tokenKeyNextToNextDate="NEXT_TO_NEXT_DATE"
+    private val tokenKeyWeekendDate = "WEEKEND"
+    private val tokenKeyWeekendNextDate = "WEEKEND_NEXT_DATE"
```

Comment:
this could be made static (comapanion object) so that we dont have to repeat same code . @iamareebjamal views?
Repeated in `SearchViewModel`